math/calculator.cpp

```diff
@@ -0,0 +1,232 @@
+            std::cin >> amount_to_subtract;
+
+            std::cout << "\n";
+
+            for (int i = 1; i <= amount_to_subtract; i++) {
+                std::cout << "Type a number: ";
+                std::cin >> number;
+
+                /**
+                 * Set the first number as the main number.
+                 * Save it in the new node so it's not re-writed.
+                 */
+
+                if (i == 1) {
+                    new_node->datatype = number;
+
+                /**
+                 * If the current number is not the first one,
+                 * subtract the current one with the previous one.
+                 */
+
+                } else {
+                    new_node->datatype -= number;
+                }
+            }
+
+            std::cout << "\nThe subtraction of the numbers is: "
+                << new_node->datatype << std::endl;
+
+            pause();
+            break;
+
+        case 3:
+            std::cout << "Type the amount of numbers to multiply: ";
+            std::cin >> amount_to_multiply;
+
+            std::cout << "\n";
+
+            for (int i = 1; i <= amount_to_multiply; i++) {
+                std::cout << "Type a number: ";
+                std::cin >> number;
+
+                /**
+                 * Set the first number as the main number.
+                 * Save it in the new node so it's not re-writed.
+                 */
+
+                if (i == 1) {
+                    new_node->datatype = number;
+
+                /**
+                 * If the current number is not the first one,
+                 * multiply the current one with the previous one.
+                 */
+
+                } else {
+                    new_node->datatype *= number;
+                }
+            }
+
+            std::cout << "\nThe multiply result of the numbers is: "
+                << new_node->datatype << std::endl;
+
+            pause();
+            break;
+
+        case 4:
+            std::cout << "Type the amount of numbers to divide: ";
+            std::cin >> amount_to_divide;
+
+            std::cout << "\n";
+
+            for (int i = 1; i <= amount_to_divide; i++) {
+                std::cout << "Type a number: ";
+                std::cin >> number;
+
+                /**
+                 * Set the first number as the main number.
+                 * Save it in the new node so it's not re-writed.
+                 */
+
+                if (i == 1) {
+                    new_node->datatype = number;
+
+                /**
+                 * If the current number is not the first one,
+                 * divide the previous number with the current one.
+                 */
+
+                } else {
+                    new_node->datatype /= number;
+                }
+            }
+
+            std::cout << "\nThe division of the numbers is: "
+                << new_node->datatype << std::endl;
+
+            pause();
+            break;
```

Comment:
a `default` case **MUST** always be added

Reply:
I feel the same.